### PR TITLE
feat: add icons for blocked segments and blocked list

### PR DESCRIPTION
**Pull request by GusBaamonde**
bloqueo

look: https://cdn.fromdoppler.com/doppler-ui-library/build1191/subscriber-history.html

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

**Comment by cbernat** [comment]
Very nice, but as we've mentioned in the daily, we need this in english and spanish, this is better as an editable text. The idea was very nice though.

File: assets/img/segments-locked.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg width="142" height="18" viewBox="0 0 142 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (cbernat) [inline assets/img/segments-locked.svg:1]: This svg is not used anymore right? We should delete this if it's not used anywhere.

**Comment by GusBaamonde** [comment]
icons removed

**Pull request merged**